tests/functional/events/17-task-event-job-logs-retrieve-command.t

```diff
@@ -49,6 +49,8 @@ workflow_run_ok "${TEST_NAME_BASE}-run" \
 
 WORKFLOW_LOG_D="${RUN_DIR}/${WORKFLOW_NAME}/log"
 sed 's/^.* -v //' "${WORKFLOW_LOG_D}/scheduler/my-rsync.log" >'my-rsync.log.edited'
+sed -i -E 's/--max-size=[^ ]* //' 'my-rsync.log.edited'  # strip "retrieve job logs max size" arg
+sort -u 'my-rsync.log.edited'  # stip out duplicates (can result from PBS log file spooling)
```

Comment:
* Strip `--max-size` options which may or may not be added based on config.
* Tolerate job log retrieval retries (needed when PBS output spooling is in play).